src/constants/globalLinks.ts

```diff
@@ -18,5 +18,5 @@ export const globalLinks = {
   bugBountyLink: "/about/zetachain/bug-bounty",
   techAmbassadorProgramLink: "https://zetachain.deform.cc/TechAmbassadors",
```

Comment:
We also need to remove the tech ambassador references and links from the site. That program has been ended.